tools/pkglint/lib/rules.ts

```diff
@@ -1079,7 +1079,11 @@ export class MustHaveNodeEnginesDeclaration extends ValidationRule {
   public readonly name = 'package-info/engines';
 
   public validate(pkg: PackageJson): void {
-    expectJSON(this.name, pkg, 'engines.node', '>= 10.13.0 <13 || >=13.7.0');
+    if (pkg.json.name === 'aws-cdk-lib') {
+      expectJSON(this.name, pkg, 'engines.node', '>= 14.15.1');
```

Comment:
Not sure it needs to be *this* edgy. This might block people with "not entirely up-to-date" environments from using the product... We could require the `14.x` line, but not be too strict about it... Some early `14.0.0` would be best, IMO - like one that's known to have feature stability.